readthedocs/core/tests/test_signals.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from django.contrib.auth.models import User
+from django_dynamic_fixture import G
```

Comment:
nitpick: we don't use `django_dynamic_fixture.G` at all, it's a poorly named variable -- linting would pick up on this if we were running pre-commit on our tests. `django_dynamic_fixture.get` is also poorly named, but much better than `G`. My personal preference is `import django_dynamic_fixture as fixture; fixture.get`.